assets/styl/_modules/_navbar.styl

```diff
@@ -24,22 +29,30 @@
         padding 0
 
         a
-            display block
             padding 0 1.8rem
             color $color-base
-            text-align center
-            transition background-color .2s
+            display flex
+
+            *
+                margin auto
 
-            &:hover
-                background-color $color-bg-base
+            svg
+                width 1.5em
+                height @width
 
     li
         float left
+        height inherit
```

Comment:
why `inherit` ?

Reply:
Because it's already defined on the parent so I thought it was better to use "inherit" than write again the same value.